aat/src/aat/resources/features/F-109 CaseAccessGroup/F-109.feature

```diff
@@ -0,0 +1,50 @@
+@F-109
+Feature: F-109: 'CaseAccessGroup' Base Complex Type
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+ @S-109.1 #AC01
+  Scenario: Must successfully import a definition file that contains CaseAccessGroup fields
+    Given a user with [an active profile in CCD],
+    When a request is prepared with appropriate values,
+    And the request [contains correctly configured CaseAccessGroup Fields],
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store],
+    Then a positive response is received,
+    And the response has all other details as expected
+
+  @S-109.2 #AC02
+  Scenario: A new case is created with CaseAccessGroup base type
+  Given a user with [an active profile in CCD],
+    When a request is prepared with appropriate values
+   And the request [contains id of a case type with CaseAccessGroup fields with caseAccessGroupType and caseAccessGroupId],
+    And it is submitted to call the [Import definition file] operation of [CCD Definition Store]
+  Then a positive response is received,
+    And the response [contains all details of the case type requested]
+    #And a call [fetch a Case Type Schema] will get the expected response as in [S-109.2_Get_CaseType].
+
+  @Ignore @S-109.3 #AC03 WIP
```

Comment:
I think we need to finish this BDD or move it out to a new branch were it will be completed later.  
Unsure if Poojitha will write/complete these BDD tests or if Devs need to.